src/components/elements/Input/FormTextArea.jsx

```diff
@@ -1,5 +1,3 @@
-/* eslint-disable jsx-a11y/label-has-associated-control */
-// TODO: why doesn't the above rule handle wrapped textarea properly?
```

Comment:
Not sure I understand why this was removed. You said you were seeing an eslint error and somehow removing the eslint-disable fixed it?

Reply:
The node build fails with the following error, I probably just don't know the right way to fix it:

```
❯ docker-compose up --build -d django
...
 => CACHED [static_files 2/8] WORKDIR /code                                                                                                                                                       0.0s
 => CACHED [static_files 3/8] COPY package.json package-lock.json tailwind.config.js /code/                                                                                                       0.0s
 => CACHED [static_files 4/8] RUN npm install --silent                                                                                                                                            0.0s
 => CACHED [static_files 5/8] COPY ./public /code/public/
 => CACHED [static_files 6/8] COPY ./src /code/src/                                                                                                                                               0.0s
 => CACHED [static_files 7/8] WORKDIR /code/
 => ERROR [static_files 8/8] RUN npm run build                                                                                                                                                   17.8s
------                                                                                                                                                                                                 
 > [static_files 8/8] RUN npm run build:                                                                                                                                                               
#26 0.518                                                                                                                                                                                              
#26 0.518 > dearpetition@0.1.0 build                                                                                                                                                                   
#26 0.518 > react-scripts build                                                                                                                                                                        
#26 0.518 
#26 1.817 Creating an optimized production build...
#26 17.69 Failed to compile.
#26 17.69 
#26 17.69 src/components/elements/Input/FormTextArea.jsx
#26 17.69   Line 1:1:  Definition for rule 'jsx-a11y/label-has-associated-control' was not found  jsx-a11y/label-has-associated-control
#26 17.69 
#26 17.69 Search for the keywords to learn more about each error.
#26 17.69 
#26 17.69 
------
executor failed running [/bin/sh -c npm run build]: exit code: 1
ERROR: Service 'django' failed to build : Build failed
```